packages/consumer/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "@chainsafe/eth-sso-consumer",
```

```suggestion
  "name": "@chainsafe/eth-sso-sdk",
```

Maybe it would be better to call this SDK or "core"?

---

Or is the intention of this package something else?